docs/articles/azure/azs-how-create-key-vault-ps.md

```diff
@@ -0,0 +1,97 @@
+---
+title: How to create a Key Vault - PowerShell | UKCloud Ltd
+description: Provides help for creating a Key Vault on UKCloud for Microsoft Azure
+services: azure-stack
+author: Bailey Lawson
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: How to create a Key Vault - PowerShell
+toc_fullpath: Users/How To/azs-how-create-key-vault-ps.md
+toc_mdlink: azs-how-create-key-vault-ps.md
+---
+
+# How to create a Key Vault in Azure Stack with PowerShell
+
+## Overview
+
+Key Vault in Azure Stack helps safeguard cryptographic keys and secrets that cloud applications and services use, by using Key Vault, you can encrypt keys and secrets.
+
+The following process will show you how to setup a vault within Azure Stacks Key Vault, store a secret in the vault and how to extract the secret.
+
+### Prerequisites
+
+Ensure your Powershell environment is setup as detailed in [Configure PowerShell Environment and Azure Stack Modules](https://docs.ukcloud.com/articles/azure/azs-how-configure-powershell-users.html) 
+
+## Creating a new Key Vault
+
+>[!Important]
+>Values to be replaced:
+>
+>Resource Group Name: <form oninput="result.value=name.value" id="resourcegroup" >

```

Comment:
I think there's something not quite right with how this is rendering. May need to escape some characters so that everything displays correctly

Reply:
The problem here lies in the fact that Github flavoured markdown doesn't support <input> tags. This has been tested on a private docFX page and works.